updated link for Citizen Code of Conduct (#959)
rust-lang/rustc-dev-guide: CODE_OF_CONDUCT.md

# The Rust Code of Conduct

> The Code of Conduct for this repository [can be found online](https://www.rust-lang.org/conduct.html).

Reviewer: ```suggestion
This repository follows the [Code of Conduct](https://www.rust-lang.org/conduct.html) of the Rust organization.
```
